### PR TITLE
Data mixer

### PR DESCRIPTION
**Pull request by tomgco**
:cocktail: :shaved_ice: 

This PR mixes in tiles and packages.

### CODE REVIEW

**Comment by tomgco** [comment]
Can someone give me a heads up on how to write some tests for this?

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "coverage": "./node_modules/.bin/babel-node ./node_modules/.bin/babel-istanbul cover ./node_modules/.bin/_mocha ./test/**/*.test.js ./lib/**/*.test.js --report lcov -- -R spec",
     "clean": "rm -rf public && mkdir -p public/",
     "build": "npm run clean && ./node_modules/.bin/webpack --optimize-minimize --optimize-occurrence-order --optimize-dedupe --config ./webpack.production.config.js",
+    "json": "npm run clean && ./node_modules/.bin/webpack --optimize-minimize --optimize-occurrence-order --optimize-dedupe --config ./webpack.production.config.js --json > public/stats.json",
```
Comment (lennym) [inline package.json]: `npm run build -- --json > public/stats.json` for brevity?

Comment (tomgco) [inline package.json]: Yes!

**Comment by nikhilaravi** [comment]
@tomgco sorry for the late reply! are you still looking at how to test this? Can tell you what we use for testing redux actions at the moment if that would help?

**Comment by tomgco** [comment]
> @tomgco sorry for the late reply! are you still looking at how to test this? Can tell you what we use for testing redux actions at the moment if that would help?

Yeah I am, If you could point me at some example tests it could help.

**Comment by tomgco** [comment]
As a note when destination tiles come in, we might want to update the ordering of these tiles.

**Pull request merged**